src/components/Person/Person.jsx

```diff
@@ -1 +1,9 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    {person.isMarried ? <p className="Person__partner">{`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}</p> : 'I am not married'}
```

Comment:
Try to avoid nesting ternary operator, instead you can create a separate variable `partner` and move this part - person.sex === 'f' ? 'husband' : 'wife' to it. Also you can use only one tag `<p>` and change only content